content/mobile-tor/contents.lr

```diff
@@ -0,0 +1,222 @@
+Orbot uses Tor to encrypt your Internet traffic. Then you can use it with other apps installed on your mobile device to
+circumvent censorship and protect against surveillance.
+Orbot can be downloaded and installed from [Google Play](https://play.google.com/store/apps/details?id=org.torproject.android).
+Check out [our Support portal](https://support.torproject.org/tormobile/tormobile-6/) to know if you need both Tor Browser for Android and Orbot or either one.
+
+
+### DOWNLOADING AND INSTALLATION
+
+There exists Tor Browser for Android and Tor Browser for Android (alpha).
+Non-technical users should get Tor Browser for Android, as this is stable and less prone to errors.
+Tor Browser for Android is available on Play Store, F-droid and the Tor Project website.
+It is very risky to download Tor Browser outside of these three platforms.
+
+#### Google Play
+You can install Tor Browser for Android from [Google Play Store](https://play.google.com/store/apps/details?id=org.torproject.torbrowser).
+
+
+#### F-Droid
+The Guardian Project provides Tor Browser for Android on their F-Droid repository.
+If you would prefer installing the app from F-Droid, please follow these steps:
+
+1. Install the F-Droid app on your Android device from [the F-droid website.](https://f-droid.org/)
+
+2. After installing F-Droid, open the app.
+
+3. At the lower-right-hand corner, open "Settings".
+
+4. Under the "My Apps" section, open Repositories.
+
+5. Toggle "Guardian Project Official Releases" as enabled.
+
+6. Now F-Droid downloads the list of apps from the Guardian Project's repository (Note: this may take a few minutes).
+
+7. Tap the Back button at the upper-left-hand corner.
+
+8. Open "Latest" at the lower-left-hand corner.
+
+10. Open the search screen by tapping the magnifying glass at the lower-right side.
+
+11. Search for "Tor Browser for Android".
+
+12. Open the query result by "The Tor Project" and install.
+
+
+#### The Tor Project website
+You can also get Tor Browser for Android by downloading and installing the apk from the [Tor Project website](https://www.torproject.org/download/#android).
+
+
+### RUNNING TOR BROWSER FOR ANDROID FOR THE FIRST TIME
+
+When you run Tor Browser for the first time, you will see the  option to connect directly to the Tor network,
+or to configure Tor Browser for your connection with the settings icon.
+
+#### Connect
+
+<img style="max-width:300px" class="col-md-6" src="../../static/images/android-connect.png" alt="Connect to Tor Browser for Android">
+
+In most cases, choosing "Connect" will allow you to connect to the Tor network without any further configuration.
+Once clicked, some text will appear, showing Tor’s connection progress.
+If you are on a relatively fast connection, but this text seems to get stuck at a certain point,
+see the [Troubleshooting](https://tb-manual.torproject.org/troubleshooting) page for help solving the problem.
```

Comment:
I found this sentence confusing: "If you are on a relatively fast connection, but this text seems to get stuck at a certain point,".  Is "a relatively fast connection" related to "text seems to get stuck"?

Reply:
Hi @sysrqb. This means that your connection may be fast, but the app doesn't connect. In that case, something may be wrong, needing troubleshooting. I actually used the choice of words directly from the manual: https://tb-manual.torproject.org/running-tor-browser/